### ISSUE TITLE
NoEcho password warning on a Lambda Function named AccountPassword

### ISSUE THREAD

**Issue by joehillen** [aws-cloudformation/cfn-python-lint#1207]
*cfn-lint version: 0.25.1*

*Description of issue.*

I'm getting this warning:

```
W2501 Inappropriate map found for password on Resources/AccountPassword
```

on this Resource in my SAM template:

```yaml
  AccountPassword:
    Type: AWS::Serverless::Function
    Properties:
      FunctionName: !Sub ${RepositoryName}-account-password-${StageName}
      Runtime: nodejs10.x
      Layers:
        - !Ref LambdaDependencyLayer
      CodeUri: dist/account
      Handler: password.handler
      Description: Reset User Password
      Timeout: 20
      Events:
        GET:
          Type: Api
          Properties:
            RestApiId:
              Ref: RestApi
            Path: /account/password
            Method: get
      Role: !GetAtt LambdaExecutionRole.Arn
    DeploymentPreference:
      Alarms:
        - !Ref AccountPasswordLatestVersionErrorMetricGreaterThanZeroAlarm
```

I'm guessing the resolution would be to make sure this check is only done on Parameters and not Resouces.

**Comment by joehillen** [comment]
@kddejong Thank you!
